src/action/interactive.rs

```diff
@@ -464,9 +463,9 @@ impl UserPicked {
 
         trace!("Select the ones to actully use");
 
-        for (path, suggestions) in suggestions_per_path {
+        for (origin, suggestions) in suggestions_per_path {
             let count = suggestions.len();
-            println!("Path is {} and has {}", path.display(), count);
+            println!("Path is {:?} and has {}", &origin, count);
```

Comment:
It has `.as_path()` which can be used with `.display()`, but it would be helpful to impl `fmt::Display` for the case we do not have an associated path in the future. Feel free to take ownership :)

Reply:
de `fmt::Display` just does the `.as_path().display()` for now, so it's extensible for the future.